ui/plugin-lifecycle/src/main/services/tenant.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from "@angular/core";
+import { Tenant, UiTenantResponse } from "../interfaces/Tenant";
+import { Observable, BehaviorSubject } from "rxjs";
+import { PluginManager } from "./plugin-manager.service";
+import { VcdApiClient } from "@vcd/sdk";
+
+@Injectable()
+export class TenantService {
+    private _orgs: Tenant[] = [];
+    private _orgsSubject = new BehaviorSubject<Tenant[]>(this.orgs);
+
+    constructor(
+        private pluginManager: PluginManager,
```

Comment:
There's no need to couple PluginManager to TenantService just to share the baseUrl.  baseUrl doesn't even seem to be necessary in this class.

Reply:
Plugin manager service is removed from tenant service in last commit.